dice-engine/dice-engine.iml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Is this file still needed?

Reply:
Missed this. 
It's not needed. I've removed the file and updated .gitignore